package.json

```diff
@@ -35,6 +35,7 @@
     "chai": "^4.1.0",
     "eslint": "^4.3.0",
     "eslint-config-shellscape": "^1.0.0",
+    "express": "^4.15.4",
```

Comment:
The issue you references was specifically for koa+websockets. It's been commonplace for koa to support https and it's demo'd in their documentation.